README.md

```diff
@@ -1,9 +1,9 @@
 # Rematch2 - Pattern-matching for Julia
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://gafter.github.io/Rematch2.jl/stable/)
-[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://gafter.github.io/Rematch2.jl/dev/)
-[![Build Status](https://github.com/gafter/Rematch2.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/gafter/Rematch2.jl/actions/workflows/CI.yml?query=branch%3Amain)
-[![Coverage](https://codecov.io/gh/gafter/Rematch2.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/gafter/Rematch2.jl)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaServices.github.io/Rematch2.jl/stable/)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaServices.github.io/Rematch2.jl/dev/)
+[![Build Status](https://github.com/JuliaServices/Rematch2.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/JuliaServices/Rematch2.jl/actions/workflows/CI.yml?query=branch%3Amain)
```

Comment:
could these repo move/license changes be a standalone PR?

Reply:
As annoying as it is, I already have a tangled mess of PRs piled on top of one another, and I'd rather not try to reorganize them.